addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/httputils/HtmlContextAnalyser.java

```diff
@@ -197,6 +197,9 @@ public List<HtmlContext> getHtmlContexts(
                         context.setTagAttribute(att.getName());
                         context.setInUrlAttribute(this.isUrlAttribute(att.getName()));
                         context.setInScriptAttribute(this.isScriptAttribute(att.getName()));
+                    } else if (att.getName().equalsIgnoreCase(target)
```

Comment:
This should have tests as well.

Reply:
This was for the eyecatcher reflection. How should I be adding a separate test for it?

Reply:
These would be like other unit tests, check that the resulting `HtmlContext` has the expected attributes.